class/defaults.yml

```diff
@@ -1,3 +1,11 @@
 parameters:
   nfs_client_provisioner:
-    namespace: syn-nfs-client-provisioner
+    namespace: syn-storage
+    server: nfs.local
+    path: /srv/nfs/export
+    mountOptions: null
+    storageClass:
```

Comment:
Please use the [component-storageclass](github.com/projectsyn/component-storage-class/) to manage storageclasses (and set `create: false` for the chart)

Reply:
that's gonna get really confusing for users, but ok